meson_options.txt

```diff
@@ -132,6 +132,9 @@ option('test-stdin', type: 'boolean', value: false,
        description: 'Enable tests that use stdin. This only works on a few targets')
 option('fortify-source', type: 'combo', choices: ['none', '1', '2', '3'], value: '3',
        description: 'Set _FORTIFY_SOURCE=<value> when building tests')
+option('test-machine', type: 'combo', choices: ['qemu', 'fvp'], value: 'qemu',
```

Comment:
Let's just make this a string type -- we'll have to do special error checking after we figure out what machines are available anyways.